__tests__/integration.test.ts

```diff
@@ -77,6 +79,30 @@ describe('when called with a GraphQL query', () => {
   })
 })
 
+describe('when called with a custom header containing wrong auth', () => {
+  beforeEach(() => {
+    process.env['INPUT_URL'] = 'https://api.github.com'
+    process.env['INPUT_HEADERS'] =
+      '{"content-type":"application/json", "Authorization": "Basic YWxhZGRpbjpvcGVuc2VzYW1l"}'
```

Comment:
Some people actually try to scrape tokens out of GH. Let's just be clear that this is a dummy value.
I'm cool with whatever name you want 🙌🏽
```suggestion
      '{"content-type":"application/json", "Authorization": "Basic pho-bac"}'
```

Reply:
changed it to : DummyToken123